hathor/transaction/block.py

```diff
@@ -312,6 +325,12 @@ def verify_weight(self) -> None:
             raise WeightError(f'Invalid new block {self.hash_hex}: weight ({self.weight}) is '
                               f'smaller than the minimum weight ({block_weight})')
 
+    def verify_height(self) -> None:
```

Comment:
Shouldn't we get the `min_height` of the tx parents and compare with this block's height?